### PR TITLE
Move bunching test changes into the dev branch (better name)

### PR DESCRIPTION
**Pull request by needmoney90**

### CODE REVIEW

**Comment by needmoney90** [comment]
@15chrjef i'll wait on your approval for this

**Comment by 15chrjef** [comment]
I approve, but it does not look like i have the ability to submit anything formal

**Review by mark-antony1 — APPROVED** [review]

**Pull request merged**
